### DISCUSSION TITLE
如何在非吉他轨道关闭六线谱？How do I turn off the six line score on a non-guitar track?

### DISCUSSION THREAD

**Discussion by isliuyaoyao** [CoderLine/alphaTab#599] [Q&A]
How do I turn off the six line score on a non-guitar track?

如何在非吉他轨道关闭六线谱？

**Comment by lannka** [comment]
Did you try setting.notation.display.staveProfile ?

**Reply by hhly** [reply]
请问下 alphatab在safari和手机上没声音是啥问题，没有报错信息

**Comment by Danielku15** [comment]
@isliuyaoyao  I think you will need to provide more details to your question than just this one liner. AlphaTab usually detects the right staves to be shown for the individual tracks. It highly depends on your setup how to hide the guitar tabs in the right way. 

1. There is the setting that @lannka suggested, there you can force the rendering of all tracks to not show any guitar tabs. 
2. But also the different file formats can contain hints whether tabs should be shown or not. While this setting is usually filled from the file readers, it can also be [overridden](https://github.com/CoderLine/alphaTab/blob/08d82858b5a4319a5ebfda76888b4dfaf8bc2bbe/src/model/Staff.ts#L82) on the [Staff level](https://alphatab.net/docs/reference/score#staff-scoretracksstaves) of the data model. 
3. If you use alphaTex there are also settings for this via the [`\staff` metadata](https://alphatab.net/docs/alphatex/tracks-staves#staves).
